lib/iob.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+function calcTotal(treatments, profile, time) {
+
+  console.info("trying to calc");
+
+  var iob = 0
+    , activity = 0;
+
+  if (!treatments) return {};
+
+  if (typeof profile === 'undefined') {
```

Comment:
Just curious, is this any different from `if (profile === undefined) {` (which is more readable)?